### PR TITLE
Add Notebook about DV Product Retrieval

### PR DESCRIPTION
**Pull request by mustaric**
This notebook describes how to use astroquery to retrieve a set of DV products for one star, L98-59. @ceb8 @scfleming

### CODE REVIEW

File: notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+    "results = parse_manifest(manifest)\n",
+    "print(results['index','sectors','fileType'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plot the DVT File\n",
+    "The time series data use to find each transit signal (or Threshold Crossing event (TCE)) is found in the dvt.fits files.  As you can see there is a dvt file. If we want the file with the most data, we should pick the one with the longest sector range in the file name.  In this case it is s0001-s0013. "
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb]: use -> used
Suggest rewording the TCE language as:
each transit signal (or Threshold Crossing event (TCE)) -> each Threshold Crossing Event (TCE), the putative transit signal,

File: notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+   "source": [
+    "print(results['index', 'sectors', 'fileType'][results['sectors'] == \"s0001-s0013\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot the entire DV median detrended, time series\n",
+    "The median detrended fluxes are stored in the first extension under 'LC_DETREND'. Notice in the plot before that L98-59, while in the continuous viewing zone, was not observed during every sector."
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb]: "Notice in the plot before" what plot are you referring to? If you mean the one after this cell, remove the word "before," if there is supposed to have already been a plot, it seems like it got deleted somewhere along the way.

File: notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+    "Each extension of the DVT data file contains a separate TCE. After the pipeline finds a set of transits, the transits are removed and the light curve is once again searched for transits.  L98-59 has three TCEs, each is consistent with the three confirmed planets found around this star.  Here we plot the phase folded light curve for each TCE, each as its own subplot.  The DVT file also contains a transit model as one of the columns in the FITS table. We overplot that in orange."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_folded(phase, data, model, ext, period):\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb]: This is a really cool plot!

File: notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+    "<br>Susan E. Mullally, STScI \n",
+    "<br>**Updated On:** 2019-09-13"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Top of Page](#title_ID)\n",
+    "<img style=\"float: right;\" src=\"./stsci_pri_combo_mark_horizonal_white_bkgd.png\" alt=\"stsci_pri_combo_mark_horizonal_white_bkgd\" width=\"200px\"/> "
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb]: Use this image call to get the logo to show up properly:
`STScI logo `

File: notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+   "metadata": {},
+   "source": [
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Input Statements"
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb]: Input -> Import

File: notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb (hunk 1 above)
Comment (mustaric) [reply notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb]: Not sure why, but I don't like the work putative.  See if what I wrote instead is clear.

File: notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+    "import numpy as np\n",
+    "from copy import deepcopy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use Astroquery to find Observations of L98-59\n",
+    "We begin by doing a cone search usig the `Observations.query_object` function and then look at just those observations made by TESS that are time series observations. In this way we should get back just those that are TESS 2-minute cadence data observations. "
```
[the remark this answers is not in the document]
Comment (ceb8) [reply notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb]: usig -> using

File: notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb (hunk 1 above)
Comment (ceb8) [reply notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb]: Heh, I overuse putative so much.  What you wrote is clear.

File: notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb [hunk 7]
```diff
@@ -0,0 +1,301 @@
...
+   "source": [
+    "# Beginner: Retrieve TESS Data Validation Products with Astroquery\n",
+    "\n",
+    "TESS (Transiting Exoplanet Survey Satelite) searches the 2-minute target data for transiting exoplanets. These data products are called DV (Data Validation) prodcuts. For every star in the search that produces a significant transiting event, a set of Data Validation products are created. The set of products are:\n",
+    "\n",
+    "- DV reports (pdf), one per star\n",
+    "- DV summaries (pdf), one per transit signal found\n",
+    "- DV mini-reports (pdf), one per star\n",
+    "- DV time series files (fits),  one per star, one extension per transit signal\n",
+    "\n",
```
[the remark this answers is not in the document]
Comment (mustaric) [reply notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb]: Ah yes, but there is only one per star within the sectors searched, which is written in the sentence before.

[the remark this answers is not in the document]
Comment (mustaric) [reply notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb]: I added something about multi-sector data up at the top.

File: notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb [hunk 8]
```diff
@@ -0,0 +1,301 @@
...
+   "source": [
+    "data_products = Observations.get_product_list(observations[obs_wanted])\n",
+    "\n",
+    "files_wanted = (data_products[\"productSubGroupDescription\"] == \"DVT\") | \\\n",
+    "                (data_products[\"productSubGroupDescription\"] == \"DVM\") | \\\n",
+    "                (data_products[\"productSubGroupDescription\"] == \"DVS\") | \\\n",
+    "                (data_products[\"productSubGroupDescription\"] == \"DVR\")\n",
+    "\n",
+    "print(data_products[\"productFilename\"][files_wanted])\n",
+    "manifest = Observations.download_products(data_products[files_wanted])\n"
```
[the remark this answers is not in the document]
Comment (mustaric) [reply notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb]: Oh, I did not know that function existed. Nice.
